packages/mockotlpserver/lib/printers.js

```diff
@@ -153,8 +156,63 @@ class JSONPrinter extends Printer {
     }
 }
 
+/**
+ * This printer converts to a possible JSON representation of each service
+ * request and saves it to a file. **Warning**: Converting OTLP service requests to JSON is fraught.
+ */
+class FilePrinter extends Printer {
```

Comment:
note for reviewer: detaching the printer from the UI service makes sense. A possible scenario is to store traces for further analysis with other tools. Also changing the usage of `jsonStringifyTrace` makes the payload more compact so files are smaller in size